pkg/session/upgrade.go

```diff
@@ -466,8 +466,13 @@ const (
 	version251 = 251
 
 	// version 252
-	// Update FSP of mysql.bind_info timestamp columns to microsecond precision.
+	// Update FSP of mysql.bind_info timestamp columns to microsecond precision and
+	//
```

Comment:
Nitpick: this change is unnecessary.
```suggestion
	// Update FSP of mysql.bind_info timestamp columns to microsecond precision.
```